src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -158,6 +158,7 @@ def create_guid(self):
 
     def cmd(self, command, checks=None, expect_failure=False):
         command = self._apply_kwargs(command)
+        print(command)
```

Comment:
should we support 2.7 currently?

Reply:
Oh. Sorry. It's just for debugging. I forgot resetting it.